app/models/folio/checkout.rb

```diff
@@ -106,6 +106,11 @@ def patron_key
       record.dig('details', 'proxyUserId') || record.dig('details', 'userId')
     end
 
+    def proxy_checkout?
+      # return true if proxyUserId exists
```

Comment:
Would you mind moving this comment before the method and perhaps use YARD doc (e.g. `# @returns true if proxyUserId exists?`)